readthedocs/rtd_tests/fixtures/spec/v2/schema.yml

```diff
@@ -53,22 +53,30 @@ python:
   # Default: '3'
   version: enum('2', '2.7', '3', '3.3', '3.4', '3.5', '3.6', required=False)
 
+  # Give the virtual environment access to the global site-packages dir
+  # Default: false | project config
```

Comment:
I'm thinking is never defaulting to a project config on v2.

Reply:
What do you mean by that? Are we not using the project model config here?

Reply:
We are, but I think the purpose of v2 is to replace all these configurations from the web admin, that way we can deprecate the admin panel faster.

Reply:
I'm doing a chagelog/guide to help the migration from v1 or web https://github.com/rtfd/readthedocs.org/pull/4451. We have discussed about suggesting a v2 config file, but we never created an issue. I'm creating one right now.